cgo/lapack_bench_test.go

```diff
@@ -0,0 +1,35 @@
+// Copyright ©2015 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package cgo
+
+import (
+	"testing"
+
+	"github.com/gonum/lapack/testlapack"
+)
+
+func BenchmarkDbdsqr100x100(b *testing.B) {
```

Comment:
Rather than this being explicit, we probably want size constants in BLAS. This way we can 

I'm not convinced 200, 300, 400  is sufficient for benchmarking.I'd rather see something like  10, 50, 100, 200, 1000. One important factor in other lapack codes is the blocking, which is typically of size 32 or 64, so having sizes that branch those breaking points is good.